certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -2311,7 +2564,7 @@
         }
       }
     },
-    "WebhookEnvelopeResponseV2": {
+    "WebhookEnvelopeResponseV3": {
```

Comment:
No need to introduce "WebhookEnvelopeResponseV3", since the response schema is not changing. We can just use WebhookEnvelopeResponseV2.

Reply:
Removed the code related to WebhookEnvelopeResponseV3